src/commands/utility/pr-template.js

```diff
@@ -1,21 +1,28 @@
-const { SlashCommandBuilder, ChannelType } = require('discord.js');
+const { SlashCommandBuilder } = require('discord.js');
 const { createPRModal } = require('../../modals/pr-template-modal');
+const { PR_TEMPLATE_ALLOWED_CHANNELS } = require('../../config');
 
 module.exports = {
   data: new SlashCommandBuilder()
     .setName('pr-template')
-    .setDescription('Create a formatted PR review request')
-    .addChannelOption((option) =>
+    .setDescription('Send a formatted PR review request')
+    .addStringOption((option) =>
       option
         .setName('channel')
-        .setDescription('Select channel to send the PR review request')
+        .setDescription('Select a channel for the PR review request')
         .setRequired(true)
-        .addChannelTypes(ChannelType.GuildText)
+        .addChoices(
+          ...PR_TEMPLATE_ALLOWED_CHANNELS.map((channel) => ({
+            name: channel.name,
+            value: channel.id,
+          }))
```

Comment:
You could do this map in the parseAllowedChannels function instead:

https://github.com/TeamNovaSoft/discord-bot/pull/68/files#r1890363392

Reply:
completed